recipes/ccache/all/conanfile.py

```diff
@@ -79,35 +88,43 @@ def package_id(self):
         del self.info.settings.compiler
 
     def build_requirements(self):
-        if hasattr(self, "settings_build") and tools.cross_building(self) and \
+        self.build_requires("pkgconf/1.9.3")
+        if hasattr(self, "settings_build") and cross_building(self) and \
```

Comment:
Why only when cross-building? I have Mac M1 so I need a newer cmake version too, but I'm not cross-building.